### ISSUE TITLE
RoosterCommandBar tooltips are not hoverable using mouse

### ISSUE THREAD

**Issue by ctmayn** [microsoft/roosterjs-react#131]
According to Microsoft accessibility requirements the tooltip itself should be able to be hovered over: 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/doc2.aspx?sourcedoc=%7B60121EB2-D1C5-4525-931F-322A940799EF%7D&file=MAS%201.4.13%20%E2%80%93%20Content%20on%20Hover%20or%20Focus.docx&action=default&mobileredirect=true&cid=bbaf7751-d8e8-4687-8f38-04965eacef9a

**Comment by JiuqingSong** [comment]
This repo is archived now. For react wrapper of roosterjs, please refer to [roosterjs-react](https://github.com/microsoft/roosterjs/tree/master/packages-ui) package in [roosterjs](https://github.com/microsoft/roosterjs) repo. Thanks.
